File: Python/1043.py
a, b, c = map(float, input().split())

if a + b > c and a + c > b and b + c > a:
    print("Perimetro = {:.1f}".format(a + b + c))
else:
    print("Area = {:.1f}".format((a + b) * c / 2))

File: Dart/1043.dart
import 'dart:io';

void main() {
  var n = stdin.readLineSync().split(' ').map(double.parse).toList();
  if (n[0] + n[1] > n[2] && n[0] + n[2] > n[1] && n[1] + n[2] > n[0]) {
    print('Perimetro = ${n[0] + n[1] + n[2]}');
  } else {
    print('Area = ${(n[0] + n[1]) * n[2] / 2}');
  }
}
